app/src/main/java/ch/epfl/polychef/pages/LoginPage.java

```diff
@@ -55,23 +44,20 @@ public void createSignInIntent(View view) {
                 RC_SIGN_IN);
     }
 
-    @Override
-    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-        super.onActivityResult(requestCode, resultCode, data);
-        FirebaseUser user = getUser();
-        if (requestCode == RC_SIGN_IN && resultCode == RESULT_OK && user != null) {
-            prepareNextActivity();
-        } else {
-            Toast.makeText(this, getString(R.string.ErrorOccurred), Toast.LENGTH_LONG).show();
-        }
-    }
-
+    /**
+     * Gets the firebase authentication current user instance.
```

Comment:
Maybe "Gets the current authenticated user from FirebaseAuth"?

Reply:
👍